### PR TITLE
Rewrite "Member Registration and Login" for V14 (and without macros)

### PR DESCRIPTION
**Pull request by kjac**
## Description

This is a complete redo of the [Member Registration and Login](https://docs.umbraco.com/umbraco-cms/tutorials/members-registration-and-login) tutorial:

- Using partial views instead of macros.
- With corrected (and less invasive) code samples.
- With updated screenshots.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

V14

## Deadline (if relevant)

When possible.

### CODE REVIEW

**Comment by jonat123** [comment]
@kjac Thanks for creating this PR! 😄

We will review it ASAP! 😄

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 1]
```diff
@@ -4,71 +4,120 @@ description: >-
...
+```
 
-Make sure to create this Macro view from available snippets.
+Complete the installer and login to the backoffice.
 
-![Partial macro snippets](images/v8-02-Partial-Macro-Snippets.png)
+## Create partial views for Registration and Login
 
-Start with the Register Member snippet.
+Navigate to the Settings section in the backoffice. Locate the Partial Views entry under Templating in the left hand section, and click the "+" to create a new partial view.
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:29]:
```suggestion
1. Navigate to the Settings section in the backoffice.
2. Locate the Partial Views entry under Templating in the left-hand section.
3. click the "+" to create a new partial view.
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 2]
```diff
@@ -4,71 +4,120 @@ description: >-
...
 
-![Partial macro register](images/v8-03-Partial-Macro-Register.png)
+In the dialog that follows, choose _New partial view from snippet..._:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:31]:
```suggestion
 4. Choose _New partial view from snippet..._: in the dialog that follows
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 3]
```diff
@@ -4,71 +4,120 @@ description: >-
...
 
-Make sure to fill out the name of the partial view (you can call it "Register", for example). Afterward, using the same approach, create a Macro Partial View using the Login snippet and name it "Login".
+![Create a new partial view](images/v14-create-partial-view-from-snippet.png)
 
-With that done, navigate to the Partial Views folder above Partial View Macro Files, and create a New partial view from the snippet. This view should be using the Login Status snippet.
+Then pick the _Login_ snippet in the next dialog:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:35]:
```suggestion
5. Pick the _Login_ snippet in the next dialog.
``.

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 4]
```diff
@@ -4,71 +4,120 @@ description: >-
...
 
-![Partial snippets](images/v8-04-Partial-Snippets.png)
+![Create a new partial view from the Login snippet](images/v14-create-partial-view-from-login-snippet.png)
 
-These three partials already contain all the necessary code to make the register, login, and logout functionality work.
+Lastly, name the partial view "Login" and save it:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:39]:
```suggestion
6. Name the partial view "Login" and save it:
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 5]
```diff
@@ -4,71 +4,120 @@ description: >-
...
 
-Since the Register and Login were created as macro partials, you will have corresponding Macros with the same names available under the Macros folder above the Templating section. Make sure to expand the Macros view, and in Editor Settings enable log in and Register macros to be used in Richtext Editor and the grid.
+![Edit and save the "Login" partial view](images/v14-create-login-partial-view.png)
 
-![Macro settings](images/v8-05-Macros-Settings.png)
+Repeat these steps using the _Register Member_ and _Login Status_ snippets. Save these partial views as "Register" and "LoginStatus", respectively.
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:43]:
```suggestion
Repeat the above steps using the _Register Member_ and _Login Status_ snippets. Save the partial views as "Register" and "LoginStatus", respectively.
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 6]
```diff
@@ -4,71 +4,120 @@ description: >-
...
+{% endcode %}
 
-![Backoffice members](images/v8-09-Backoffice-Members.png)
+## Create the Register/Login page
 
-Now, we have our signup and login functionality - though there is one partial view we have not used yet, the Login Status. We created it as a Partial View, not as a Macro, for a singular reason - we are going to include it directly in a template, not in a property editor in the Content section. This partial view will show the current status - whether the person is logged in or not.
+Halfway there!
 
-To insert the partial, type `@Html.Partial("")` and enter the name of your partial view in the quotes.
+Next, navigate to the Content section. Under the _Home_ node, create a new page based on the "Login" Document Type:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:104]:
```suggestion
1. Navigate to the Content section. 
2. Create a new page based on the "Login" Document Type Under the _Home_ node: 
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 7]
```diff
@@ -4,71 +4,120 @@ description: >-
...
 
-A good place to put it would be either the Master page or a navigation component.
+![Creating the Register/Login page](images/v14-create-register-login-page.png)
 
-![Login partial in navigation](images/v8-10-Login-Partial-In-Nav.png)
+Save and publish the page. The Register and Login functionality is rendered by the "Login" template:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:108]:
```suggestion
3. Save and publish the page. 

The Register and Login functionality is rendered by the "Login" template:
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 8]
```diff
@@ -93,265 +142,109 @@ Now that we have the options to:
 * Check the current login status
 * Log out a member
 
-We can also go a bit further and specify which parts of our website should be accessible to logged-in members. To do this, head on over to the Member section in the Backoffice, and create a new Member Group.
+We can take this a bit further and specify which parts of our website should be accessible to logged-in members. To do this, head on over to the Member section in the Backoffice, and create a new Member Group.
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:150]:
```suggestion
We can take this a bit further and specify which parts of our website should be accessible to logged-in members. To do this:
 1. Go to the Member section in the Backoffice.
 2. Create a new Member Group.
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 9]
```diff
@@ -93,265 +142,109 @@ Now that we have the options to:
...
 
-![New member group](images/v8-12-New-Member-Group.png)
+![Create a new member group](images/v14-create-member-group.png)
 
-Give the group a name, and save it. Next, move over to the Members folder, click on any of the Members you have, and assign the newly created Member Group to them. To do that, click on the member who you would like to assign to the group. Under `Properties`, you have the option to pick a group to assign the member to `(Member Group` property).
+Give the group a name and save it:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:154]:
```suggestion
3. Give the group a name.
4. Save the Member Group.
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 10]
```diff
@@ -93,265 +142,109 @@ Now that we have the options to:
...
 
-![Member group assign](images/v8-13-Member-Group-Assign.png)
+![Naming the new member group](images/v14-create-member-group-step-2.png)
 
-Almost there. Save the member, move to the Content section and find the page you would like to restrict. Right-click on that content node and find the option "Restrict Public Access".
+Next, navigate back to the created Member. Assign the newly created Member Group and save the member:
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:158]:
```suggestion
5. Navigate back to the created Member. 
6. Assign the newly created Member Group.
7. Save the member:
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 11]
```diff
@@ -93,265 +147,114 @@ Now that we have the options to:
...
-![Restrict public access](images/v8-14-Restrict-Public-Access.png)
+![Assign the new Member group to the created Member](images/v14-assign-member-group.png)
 
-You will be able to restrict access to a specific member, or a specific group. Choose the latter option. On this menu, you will be able to select the group that will have access to the page, the page with the login form, and the page that would be displayed if selected content would be inaccessible to the chosen group.
+Almost there!
 
-{% hint style="info" %}
-To restrict access you need to have made a member group, and a page containing the login partial. Having a "No access"/error page is recommended, as well - though you can use any content page you have.
-{% endhint %}
+Now navigate to the Content section. Create a new page that should only be visible to "Premium" members.
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:169]:
```suggestion
1. Navigate to the Content section. 
2. Create a new page that should only be visible to "Premium" members.
```

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 12]
```diff
@@ -93,265 +147,114 @@ Now that we have the options to:
...
 
-![Restrict public access details](images/v8-15-Restrict-Public-Access-Details.png)
+Once created, click the menu icon (•••) to bring up the page options, and pick "Restrict Public Access":
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:171]:
```suggestion
 3. Click the menu icon (•••) to bring up the page options, and pick "Restrict Public Access" once created.
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@kjac Looks good!

Just had some minor suggestions to put the steps in the tutorial into steps 😄

**Comment by kjac** [comment]
Good stuff @jonat123 - I have merged all your suggestions in 😄

File: 14/umbraco-cms/tutorials/members-registration-and-login.md [hunk 13]
```diff
@@ -4,71 +4,125 @@ description: >-
...
-Make sure to create this Macro view from available snippets.
+Complete the installer and login to the backoffice.
 
-![Partial macro snippets](images/v8-02-Partial-Macro-Snippets.png)
+## Create partial views for Registration and Login
 
-Start with the Register Member snippet.
+1. Navigate to the Settings section in the backoffice.
+2. Locate the Partial Views entry under Templating in the left-hand section.
+3. click the "+" to create a new partial view.
```
Comment (jonat123) [inline 14/umbraco-cms/tutorials/members-registration-and-login.md:31]:
```suggestion
3. Click the "+" to create a new partial view.
```

**Review by jonat123 — APPROVED** [review]

**Pull request merged**